lua/cl_spectator_deathmatch.lua

```diff
@@ -62,48 +62,43 @@ hook.Add("RenderScreenspaceEffects", "RenderScreenspaceEffects_Ghost", function(
 	end
 end)
 
+local COLOR_WHITE = COLOR_WHITE
```

Comment:
Micro optimization. This probably brings very little, but often used things should be precached. You could also just precache the color. But calling Color() the whole time is terrible.